### DISCUSSION TITLE
EJBCA CE 8 Rest API link

### DISCUSSION THREAD

**Discussion by merryo11** [Keyfactor/ejbca-ce#525] [General]
Hi All, I need to use rest api of EJBCA CE 8 so can any one tell what shall be the API link because I can only access the SOAP base API at the moment.

https://localhost/ejbca/ejbcaws/ejbcaws?wsdl (SOAP API)

Thanks

**Comment by primetomas** [comment]
You can find it all in the documentation. 
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols/ejbca-rest-interface
